recipe/meta.yaml

```diff
@@ -82,4 +82,3 @@ extra:
     - carlodri
     - adbeda
     - oblute
-    - rlura14
```

Comment:
@adbeda, if this username was a typo, please replace with the correct username and rerender.  See https://github.com/conda-forge/tesseract-feedstock/pull/21#issuecomment-598655674